srcpkgs/nyxt/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'nyxt'
+pkgname=nyxt
+version=2.0.0
+revision=1
+wrksrc=nyxt
+build_style=gnu-makefile
+hostmakedepends="sbcl curl pkg-config git"
+makedepends="webkit2gtk-devel libfixposix-devel openssl-devel libgirepository-devel"
+depends="dbus sqlite xclip libfixposix-devel openssl-devel libgirepository-devel"
+short_desc="Keyboard-oriented, extensible web-browser"
+maintainer="0x0f0f0f <sudo-woodo3@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://nyxt.atlas.engineer/"
+distfiles="https://nyxt.atlas.engineer/static/release/nyxt.tar.gz"
```

Comment:
Please use versioned tarball. Github has one.

Reply:
This is a problem because the release archive on github is broken (doesn't contain submodules) and the developer upload a fixed one here: https://nyxt.atlas.engineer/static/release/nyxt.tar.gz after i talked to him.